src/ttrssapi.cpp

```diff
@@ -416,15 +416,12 @@ rsspp::Feed TtRssApi::fetch_feed(const std::string& id, CURL* cached_handle)
 
 			if (!item_obj["attachments"].is_null()) {
 				if (item_obj["attachments"].size() >= 1) {
-					json a =
-						item_obj["attachments"].front();
+					json a = item_obj["attachments"].front();
 					if (!a["content_url"].is_null()) {
-						item.enclosure_url =
-							a["content_url"];
+						item.enclosure_url =a["content_url"];
```

Comment:
A space is missing after the assignment operator.

Reply:
Fixed in https://github.com/newsboat/newsboat/pull/854/commits/f0956cf62ed2207334c2035097d0ab4d7e17fc12.